plugin/pkg/auth/authorizer/rbac/bootstrappolicy/controller_policy.go

```diff
@@ -253,6 +253,7 @@ func buildControllerRoles() ([]rbacv1.ClusterRole, []rbacv1.ClusterRoleBinding)
 			// used for pod eviction
 			rbacv1helpers.NewRule("update").Groups(legacyGroup).Resources("pods/status").RuleOrDie(),
 			rbacv1helpers.NewRule("list", "delete").Groups(legacyGroup).Resources("pods").RuleOrDie(),
+			rbacv1helpers.NewRule("get", "list", "create", "watch", "update", "delete", "patch").Groups(networkingGroup).Resources("clustercidrconfigs").RuleOrDie(),
```

Comment:
why does node-controller need update/patch ?

Reply:
Removed the additional permissions

Reply:
this has been resolved but the `update/patch` values are still there